packages/visual-editor/src/hooks/useDropzoneDirection.ts

```diff
@@ -1,11 +1,20 @@
-import { CONTENTFUL_CONTAINER_ID } from '@contentful/experience-builder-core/constants';
+import {
+  CONTENTFUL_CONTAINER_ID,
+  CONTENTFUL_SINGLE_COLUMN_ID,
+  CONTENTFUL_COLUMNS_ID,
+} from '@contentful/experience-builder-core/constants';
 import type {
   CompositionComponentNode,
   ResolveDesignValueType,
 } from '@contentful/experience-builder-core/types';
 import { useEffect } from 'react';
 import { useZoneStore } from '@/store/zone';
 
+const isStructureComponent = (blockId: string) => {
+  return [CONTENTFUL_COLUMNS_ID, CONTENTFUL_CONTAINER_ID, CONTENTFUL_SINGLE_COLUMN_ID].includes(
+    blockId
+  );
+};
```

Comment:
I added a similar util method to the core package in my Section component POC #279 and named it `isContentfulStructureComponent`

* https://github.com/contentful/experience-builder/blob/245d2b86d392858fcc710c7d53e74a66593b4c02/packages/core/src/utils/components.ts

Reply:
Sweet, I'll use that instead

Reply:
Realized this isn't on `development` yet so I'll just make a note to refactor this section once that PR merges